src/transform/observable-quaternion.ts

```diff
@@ -78,6 +78,11 @@ export class ObservableQuaternion extends ObservablePoint {
     }
   }
 
+  /** The euler representation of the quaternion. */
+  get euler() {
```

Comment:
Maybe this should be called "getEulerAngles" just to make it clear to user that you can do "quaternion.euler.x = 5".

Reply:
fair point, will rework this today

Reply:
Thanks!